pom.xml

```diff
@@ -70,13 +70,13 @@
             <version>${workflow.version}</version>
             <optional>true</optional>
         </dependency>
+
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>run-selector</artifactId>
-            <version>1.0-alpha-2-SNAPSHOT</version>
-            <optional>true</optional>
+            <version>1.0-beta-1</version>
+            <scope>test</scope>
```

Comment:
To be discussed if this should be <scope>test</scope> or <optional>true</optional>